packages/AutoTDD-GUI.package/ATDDGitHubActionResults.class/instance/simplifySmalltalkCiLogCheckBoxState.st

```diff
@@ -0,0 +1,4 @@
+building
+simplifySmalltalkCiLogCheckBoxState
+	
+	^ self shouldSimplifySmalltalkCiLog
```

Comment:
```suggestion
accessing
simplifyLogCheckBoxState
	
	^ self shouldSimplifyLog
```
I felt the names were a little too long to support readability. Although the parsing is in the same class i would not include any info on implementation (SmalltalkCiLog vs Log) in GUI code